internal/providers/terraform/google/testdata/artifact_registry_repository_test/artifact_registry_repository_test.golden

```diff
@@ -0,0 +1,32 @@
+
+ Name                                                              Monthly Qty  Unit            Monthly Cost 
+                                                                                                             
+ google_artifact_registry_repository.asia_east1_usage                                                        
+ ├─ Storage usage                                                          150  GB                    $15.00 
+ └─ Data egress AsiaPacific to AsiaPacific                                 100  GB                     $5.00 
+                                                                                                             
+ google_artifact_registry_repository.australia_southeast1_usage                                              
+ ├─ Storage usage                                                          150  GB                    $15.00 
+ └─ Data egress from/to Oceania                                            100  GB                     $8.00 
+                                                                                                             
+ google_artifact_registry_repository.europe_north1_usage                                                     
+ ├─ Storage usage                                                          150  GB                    $15.00 
+ ├─ Data egress from/to Oceania                                            100  GB                    $15.00 
```

Comment:
Should we have `Europe to Oceania`?

Or if we want to map more to what they have on [their pricing page](https://cloud.google.com/artifact-registry/pricing) should it be `from/to Oceania` and `intercontinental (excl. Oceania)` instead of `Europe to South America`?

Reply:
yup sorry I'm unsure why I did a hybrid approach with their pricing page and a nice approach. I think `Europe to Oceania` is better 👍

Reply:
actually decided to be more inline with the pricing, changes here: https://github.com/infracost/infracost/pull/1203/commits/1a7b9526b0a2e8f496807feec272fb7dccd52659